'npm start' gives error

**This is a Bug Report**

<!-- Thanks for filing an issue! Before submitting, please fill in the following information. -->
<!-- See https://kubernetes.io/docs/contribute/start/ for guidance on writing an actionable issue description. -->

<!--Required Information-->
**Problem:**
While following [README](https://github.com/kubernetes-sigs/release-notes#development) of this repo, 
`npm start` gives error.

[How to reproduce]
```
❯ go env                                 
GO111MODULE="on"
```

```
❯ go get k8s.io/release/cmd/release-notes
(Downloads `k8s.io/release@v0.8.0`)
```

```
❯ cd go/pkg/mod/k8s.io/release@v0.8.0/
❯ ls -alh      
total 312K
dr-xr-xr-x 11 root root 4.0K Apr 20 09:52 .
drwxr-xr-x  3 root root 4.0K Apr 20 09:52 ..
dr-xr-xr-x  3 root root 4.0K Apr 20 09:52 .github
-r--r--r--  1 root root 2.2K Apr 20 09:52 .gitignore
-r--r--r--  1 root root 2.6K Apr 20 09:52 .golangci.yml
-r--r--r--  1 root root  641 Apr 20 09:52 CONTRIBUTING.md
-r--r--r--  1 root root 1.5K Apr 20 09:52 Dockerfile-kubepkg
-r--r--r--  1 root root 1.2K Apr 20 09:52 Dockerfile-kubepkg-rpm
-r--r--r--  1 root root  12K Apr 20 09:52 LICENSE
-r--r--r--  1 root root 4.4K Apr 20 09:52 Makefile
-r--r--r--  1 root root  242 Apr 20 09:52 OWNERS
-r--r--r--  1 root root 1.9K Apr 20 09:52 OWNERS_ALIASES
-r--r--r--  1 root root 4.0K Apr 20 09:52 README.md
-r--r--r--  1 root root 1.1K Apr 20 09:52 SECURITY.md
-r--r--r--  1 root root  581 Apr 20 09:52 SECURITY_CONTACTS
-r--r--r--  1 root root  923 Apr 20 09:52 cloudbuild-kubepkg.yaml
dr-xr-xr-x 12 root root 4.0K Apr 20 09:52 cmd
-r--r--r--  1 root root  148 Apr 20 09:52 code-of-conduct.md
-r--r--r--  1 root root 2.5K Apr 20 09:52 compile-release-tools
-r--r--r--  1 root root 9.0K Apr 20 09:52 dependencies.yaml
dr-xr-xr-x  3 root root 4.0K Apr 20 09:52 docs
dr-xr-xr-x  5 root root 4.0K Apr 20 09:52 gcb
-r--r--r--  1 root root 1.8K Apr 20 09:52 go.mod
-r--r--r--  1 root root 123K Apr 20 09:52 go.sum
dr-xr-xr-x  5 root root 4.0K Apr 20 09:52 hack
dr-xr-xr-x  6 root root 4.0K Apr 20 09:52 images
dr-xr-xr-x  2 root root 4.0K Apr 20 09:52 internal
dr-xr-xr-x  3 root root 4.0K Apr 20 09:52 packages
dr-xr-xr-x 27 root root 4.0K Apr 20 09:52 pkg
-r--r--r--  1 root root  56K Apr 20 09:52 push-build.sh
```

```
❯ npm install
npm WARN root No description
npm WARN root No repository field.
npm WARN root No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/fork-ts-checker-webpack-plugin/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/watchpack-chokidar2/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 1291 packages in 6.663s

57 packages are looking for funding
  run `npm fund` for details

found 11 vulnerabilities (1 low, 8 moderate, 2 high)
  run `npm audit fix` to fix them, or `npm audit` for details
```

```
❯ npm start  
npm ERR! missing script: start

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-04-20T00_55_06_868Z-debug.log

❯ npm run docker:build:run
npm ERR! missing script: docker:build:run

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-04-20T00_55_22_100Z-debug.log
```

```
❯ cd cmd/release-notes

❯ pwd            
/root/go/pkg/mod/k8s.io/release@v0.8.0/cmd/release-notes

❯ ls -alh
total 36K
dr-xr-xr-x  2 root root 4.0K Apr 20 09:52 .
dr-xr-xr-x 12 root root 4.0K Apr 20 09:52 ..
-r--r--r--  1 root root  129 Apr 20 09:52 OWNERS
-r--r--r--  1 root root 9.3K Apr 20 09:52 README.md
-r--r--r--  1 root root  11K Apr 20 09:52 main.go
```

```
❯ npm install             
npm WARN root No description
npm WARN root No repository field.
npm WARN root No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/fork-ts-checker-webpack-plugin/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/watchpack-chokidar2/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 1291 packages in 6.459s

57 packages are looking for funding
  run `npm fund` for details

found 11 vulnerabilities (1 low, 8 moderate, 2 high)
  run `npm audit fix` to fix them, or `npm audit` for details
```

```
❯ npm start  
npm ERR! missing script: start

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-04-20T00_57_07_452Z-debug.log

❯ npm run docker:build:run
npm ERR! missing script: docker:build:run

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-04-20T00_57_27_893Z-debug.log
```

---

```
❯ cd
❯ GO111MODULE=off go get k8s.io/release/cmd/release-notes
cannot find package "github.com/google/go-github/v33/github" in any of:
        /usr/local/go/src/github.com/google/go-github/v33/github (from $GOROOT)
        /root/go/src/github.com/google/go-github/v33/github (from $GOPATH)
```

```
❯ cd go/src/k8s.io/release/
❯ ls -alh
total 308K
drwxr-xr-x 12 root root 4.0K Apr 20 09:46 .
drwxr-xr-x  3 root root 4.0K Apr 20 09:35 ..
drwxr-xr-x  8 root root 4.0K Apr 20 09:40 .git
drwxr-xr-x  3 root root 4.0K Apr 20 09:35 .github
-rw-r--r--  1 root root 2.2K Apr 20 09:35 .gitignore
-rw-r--r--  1 root root 2.6K Apr 20 09:35 .golangci.yml
-rw-r--r--  1 root root  641 Apr 20 09:35 CONTRIBUTING.md
-rw-r--r--  1 root root 1.5K Apr 20 09:35 Dockerfile-kubepkg
-rw-r--r--  1 root root 1.2K Apr 20 09:35 Dockerfile-kubepkg-rpm
-rw-r--r--  1 root root  12K Apr 20 09:35 LICENSE
-rw-r--r--  1 root root 4.4K Apr 20 09:35 Makefile
-rw-r--r--  1 root root  242 Apr 20 09:35 OWNERS
-rw-r--r--  1 root root 1.9K Apr 20 09:35 OWNERS_ALIASES
-rw-r--r--  1 root root 4.0K Apr 20 09:35 README.md
-rw-r--r--  1 root root 1.1K Apr 20 09:35 SECURITY.md
-rw-r--r--  1 root root  581 Apr 20 09:35 SECURITY_CONTACTS
-rw-r--r--  1 root root 1.1K Apr 20 09:35 cloudbuild-kubepkg.yaml
drwxr-xr-x 12 root root 4.0K Apr 20 09:35 cmd
-rw-r--r--  1 root root  148 Apr 20 09:35 code-of-conduct.md
-rwxr-xr-x  1 root root 2.5K Apr 20 09:35 compile-release-tools
-rw-r--r--  1 root root  12K Apr 20 09:35 dependencies.yaml
drwxr-xr-x  3 root root 4.0K Apr 20 09:35 docs
drwxr-xr-x  5 root root 4.0K Apr 20 09:35 gcb
-rw-r--r--  1 root root 1.8K Apr 20 09:35 go.mod
-rw-r--r--  1 root root 115K Apr 20 09:35 go.sum
drwxr-xr-x  5 root root 4.0K Apr 20 09:35 hack
drwxr-xr-x  6 root root 4.0K Apr 20 09:35 images
drwxr-xr-x  2 root root 4.0K Apr 20 09:35 internal
drwxr-xr-x  4 root root 4.0K Apr 20 09:35 packages
drwxr-xr-x 28 root root 4.0K Apr 20 09:35 pkg
-rwxr-xr-x  1 root root  56K Apr 20 09:35 push-build.sh
```

```
❯ git log

commit c0d02130e78a83d0f7262cbb776cd9e9928686b5 (HEAD -> master, origin/master, origin/HEAD)
Merge: a9b36dbc 85ca452a
Author: Kubernetes Prow Robot <k8s-ci-robot@users.noreply.github.com>
Date:   Mon Apr 19 06:00:37 2021 -0700

    Merge pull request #2010 from saschagrunert/skip-note
    
    Add notice if release note gets excluded because of mismatched authors
...
```

(Under `go/src/k8s.io/release/`,
`npm start` and `npm run docker:build:run` give error too.)

**Proposed Solution:**



<!--Optional Information (remove the comment tags around information you would like to include)-->
<!--Kubernetes Version:-->

<!--Additional Information:-->

--- a/README.md
+++ b/README.md
@@ -12,7 +12,7 @@ Local requirements:
 
 - node/npm
 - golang > 1.11 (to run the release-notes JSON tool)
-- `go get k8s.io/release/cmd/release-notes`
+- `git clone https://github.com/kubernetes-sigs/release-notes.git`
   (to generate any additional notes for development)
 
 Run `npm install && npm start` from the root dir to start up the angular

